app/helpers/wally_integration.py

```diff
@@ -1,29 +1,30 @@
 import os
+import string
 import requests
 
 from app.models import Review, User, DiveShop, Spot
 
 wally_api_base = os.environ.get('WALLY_API')
 wally_auth_token = os.environ.get('WALLY_AUTH_TOKEN')
 
-def mint_nft(current_review: Review, dive_shop: DiveShop, beach: Spot, user: User):
+def mint_nft(current_review: Review, dive_shop: DiveShop, beach: Spot, user: User, signature: string):
   payload = {
```

Comment:
i think instead of passing in the signature maybe we should include the call to sign_message inside this function (as opposed to all the different controllers)